### ISSUE TITLE
Add Translation Platform Support

### ISSUE THREAD

**Issue by lucasmz-dev** [futo-org/android-keyboard#554]
### Use Cases

* Better accessibility to the rest of the world

### Proposal

Have FUTO Keyboard on a platform like Weblate

**Comment by MXC48** [comment]
Yes I am also open to improve the translation for my language (French) !
Have the application on weblate, Crowdin or transifex would be very useful !

**Comment by dean-futo** [comment]
Duplicate of #567 - merging this request into that one.

**Comment by lucasmz-dev** [comment]
Hey um this isn't about translation as a feature but the translation of the app itself

**Comment by lucasmz-dev** [comment]
Weblate has free hosting for libre projects, but FUTO Keyboard isn't one; [they specifically check for licenses approved by OSI or FSF.](https://hosted.weblate.org/hosting/)
This still means FUTO could either purchase hosting or host Weblate themselves, as it is FOSS.

**Comment by abb128** [comment]
Now available at https://i18n-keyboard.futo.org/

**Comment by lucasmz-dev** [comment]
I will say I'm not entirely sure I love Pontoon, there are a few maintenance issues I can see arising:

1. Source string changes don't seem to propagate? Some strings were just changed and it doesn't seem to be shown anywhere that the translation may need to be updated (https://github.com/futo-org/android-keyboard/commit/8f06577d7ebb63408bc463c7af3e7679e5aa5c22)

2. It seems to still show strings with the translatable tag as false on it (which led to https://github.com/futo-org/android-keyboard/commit/62a5f0d14095d1f258c19fc09e425b49b10dff9f)

Weblate deals with both of those issues and works very nicely for Android resources from my experience, with one caveat being the order of strings is lost, on the other side I had never seen Pontoon before and when I search it up I only really see it being used for Firefox

**Comment by abb128** [comment]
Thanks a lot for the translations!

I chose Pontoon over Weblate mainly because I personally prefer its user interface, I found Weblate quite cumbersome and couldn't imagine translating things with it myself. Pontoon is also used by [SlimeVR](https://i18n.slimevr.dev) which was how I came to know about it.

Source string changes does seem to be a problem, it's also really not handling Android tags well which is a bigger problem: https://gitlab.futo.org/keyboard/keyboard-translations/-/blob/master/core/values-pt-rBR/strings-uix.xml?ref_type=heads#L35

These problems can be worked around, they're not complete dealbreakers, if it provides a better translator experience through a nicer UI then I'm happy

**Comment by lucasmz-dev** [comment]
Would it be possible for me to get a CI build of the dev variant so I can go around reviewing the translation?

**Comment by abb128** [comment]
Check https://docs.keyboard.futo.org/improvements/nightly
